src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -306,19 +306,20 @@ where
     pub async fn add_monotonic_delete(&mut self, event: MonotonicDeleteEvent) -> HummockResult<()> {
         if let Some(builder) = self.current_builder.as_mut()
             && builder.reach_capacity()
-            && event.new_epoch != MAX_EPOCH
+            && !is_max_epoch(event.new_epoch)
         {
-            if builder.last_range_tombstone_epoch() != MAX_EPOCH {
+            if  !is_max_epoch(builder.last_range_tombstone_epoch()) {
                 builder.add_monotonic_delete(MonotonicDeleteEvent {
                     event_key: event.event_key.clone(),
-                    new_epoch: MAX_EPOCH,
+                    // write `HummockEpoch::MAX` instead of `HummockEpoch::MAX` for forward compatibility.
```

Comment:
fix comments ``HummockEpoch::MAX` instead of `HummockEpoch::MAX``?

Reply:
Fixed